tests/lib/prepare.sh

```diff
@@ -78,21 +78,34 @@ setup_reflash_magic() {
         # build new core snap for the image
         snapbuild $UNPACKD $IMAGE_HOME
 
-        # FIXME: remove once we have a proper model.assertion
-        . $TESTSLIB/store.sh
-        STORE_DIR=/tmp/fake-store-blobdir
-        mkdir -p $STORE_DIR
-        setup_store fake-w-assert-fallback $STORE_DIR
-        cp $TESTSLIB/assertions/developer1.account $STORE_DIR/asserts
-        cp $TESTSLIB/assertions/developer1.account-key $STORE_DIR/asserts
+        # FIXME: fetch directly once its in the assertion service
+        cat > $IMAGE_HOME/pc.model <<EOF
```

Comment:
The file should probably go into $TESTS/lib/pc-amd64.model and be copied. Okay to do that later, though.